app/views/govuk_publishing_components/components/docs/contents_list.yml

```diff
@@ -193,6 +193,22 @@ examples:
             text: 2. Numbers not parsed
           - href: "#third-thing"
             text: 3. Numbers are just text
+  with_alternate_active:
```

Comment:
It's hard naming things but maybe something more descriptive like `without_dashes` would be clearer?

Reply:
It's not just without dashes, it gets different vertical line-spacing between each option and the left hand blue vertical border for the active state. I also wanted to avoid using a name that included 'landing page'. I think this name is OK given the nature of what it does, but maybe removing the `with-` part is enough to simplify it?

Reply:
I wonder now if this means we need a different option: expanded line spacing? This would allow the expanded spacing to be used by itself without also adopting the alternate active state... It does seem a little confusing that the alternate active state is also applying a different line height.

Reply:
`alternate_active` in my head implies that every other element is active. How about `with_no_dashes_and_wider_spacing`?

Reply:
> I wonder now if this means we need an extra option: expanded line spacing? This would allow the expanded spacing to be used by itself without also adopting the alternate active state. It does seem a little confusing that the alternate active state is also applying a different line height.

I don't think we want to start adding multiple variations when we only need one - we can always split them apart later if it turns out we need one without the other.

Reply:
Hmm, but it also gets a completely different visual treatment of the active state which isn't communicated by that name. Getting complex if we add that in though!

What about simply `alternative-line-style`? This isn't explicit, but it does cover all the changes if you think about it :) The associated description then explains all the changes.

Reply:
Sure, go with that.

Reply:
Rebasing went a bit awry, but after a bit of multi-branch cherry-picking, this is now changed as discussed.